catalog/security/banner-marking/src/test/groovy/org/codice/alliance/security/banner/marking/BannerCommonMarkingExtractorSpec.groovy

```diff
@@ -319,6 +312,114 @@ class BannerCommonMarkingExtractorSpec extends Specification {
         '//COSMIC RESTRICTED//BOHEMIA' | true            | null
     }
 
+    def 'test security conflict classification'() {
+        when:
+        initBannerMarkings("TOP SECRET")
+        metacard.setAttribute(new AttributeImpl(Security.CLASSIFICATION, "CONFLICTING VALUE"))
+        validationException = null
+        try {
+            extractor.processClassMarking(metacard, bannerMarkings)
+        } catch (RuntimeException rte) {
+            validationException = rte
+        }
+
+        then:
+        validationException != null
+        validationException.getMessage() == "Extracted security attribute security.classification" +
+                " from the banner markings does not match value pre-existing on the" +
+                " metacard. The metacard will not be created."
+    }
+
+    def 'test security conflict releasability'() {
+        when:
+        initBannerMarkings("SECRET//TK//REL TO CAN, DEU, NZL")
+        metacard.setAttribute(new AttributeImpl(Security.RELEASABILITY, "CONFLICTING VALUE"))
```

Comment:
✏️ I think your tests would be more definitive and less prone to being broken in the future if you used real values. So instead of `CONFLICTING VALUE` here, use a real releasability value such as `GBR`